libs/acoustid_ext/setup.py

```
@@ -0,0 +1,18 @@
+from setuptools import setup, Extension
+from Cython.Build import cythonize
+
+extensions = [
+    Extension(
+        "acoustid_ext.fingerprint",
+        sources=["src/acoustid_ext/fingerprint.pyx"],
+        extra_compile_args=["-O3"]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)